registries/_format/binary.md

```diff
@@ -4,6 +4,8 @@ issue:
 description: any sequence of octets
 base_type: string
 layout: default
+oas_version: < 3.1
+remarks: "When using OpenAPI 3.1 or above it's recommended not to use this format and instead set the `mediaType` to `application/octet-stream`, `image/png`,... without a `schema` property. This format MUST ONLY be used for schemas that represent the whole request/response body and MUST NOT be used for properties, arrays, etc..."
```

Comment:
Streamlining:

> In OpenAPI 3.1, instead set the media type appropriately and do not use a `schema` property.  Note that only complete HTTP message bodies or complete parts in a `multipart` media type can accommodate binary data.  JSON strings and URL path components cannot.

You can link part of the text to the part of the spec that shows the example.

I'll defer to the @OAI/tsc , but I'd avoid capital MUST in the registry as it is not a normative document.

Reply:
updated